bridge_reachability/tor-bridges-ip-port.csv

```diff
@@ -0,0 +1,109 @@
+192.240.101.106,80,fdctorbridge01,fte
+198.245.60.50,443,riemann,obfs4
+198.245.60.50,22,riemann,ssh
+192.99.11.54,443,noether,obfs4
+192.99.11.54,22,noether,ssh
+154.35.22.9,60873,GreenBelt,obfs4
+154.35.22.9,1984,GreenBelt,obfs4
+154.35.22.9,443,GreenBelt,obfs4
+154.35.22.9,80,GreenBelt,obfs4
+154.35.22.9,5881,GreenBelt,obfs4
+154.35.22.9,7013,GreenBelt,obfs4
+154.35.22.9,12166,GreenBelt,obfs4
+154.35.22.9,14303,GreenBelt,obfs4
+154.35.22.9,25427,GreenBelt,obfs4
+154.35.22.9,29733,GreenBelt,obfs4
+154.35.22.9,40782,GreenBelt,obfs4
+154.35.22.9,42487,GreenBelt,obfs4
+154.35.22.9,48869,GreenBelt,obfs4
+154.35.22.9,50819,GreenBelt,obfs4
+154.35.22.10,41835,Mosaddegh,obfs4
+154.35.22.10,1984,Mosaddegh,obfs4
+154.35.22.10,443,Mosaddegh,obfs4
+154.35.22.10,80,Mosaddegh,obfs4
+154.35.22.10,2934,Mosaddegh,obfs4
+154.35.22.10,9332,Mosaddegh,obfs4
+154.35.22.10,15937,Mosaddegh,obfs4
+154.35.22.10,24338,Mosaddegh,obfs4
+154.35.22.10,26336,Mosaddegh,obfs4
+154.35.22.10,26703,Mosaddegh,obfs4
+154.35.22.10,40348,Mosaddegh,obfs4
+154.35.22.10,46345,Mosaddegh,obfs4
+154.35.22.10,55622,Mosaddegh,obfs4
+154.35.22.10,56472,Mosaddegh,obfs4
+154.35.22.10,22,Mosaddegh,ssh
+154.35.22.11,49868,MaBishomarim,obfs4
+154.35.22.11,1984,MaBishomarim,obfs4
+154.35.22.11,443,MaBishomarim,obfs4
+154.35.22.11,80,MaBishomarim,obfs4
+154.35.22.11,2413,MaBishomarim,obfs4
+154.35.22.11,7920,MaBishomarim,obfs4
+154.35.22.11,16488,MaBishomarim,obfs4
+154.35.22.11,17613,MaBishomarim,obfs4
+154.35.22.11,36652,MaBishomarim,obfs4
+154.35.22.11,44594,MaBishomarim,obfs4
+154.35.22.11,54823,MaBishomarim,obfs4
+154.35.22.11,58028,MaBishomarim,obfs4
+154.35.22.11,60166,MaBishomarim,obfs4
+154.35.22.11,64841,MaBishomarim,obfs4
+154.35.22.11,22,MaBishomarim,ssh
+154.35.22.12,80,JonbesheSabz,obfs4
+154.35.22.12,443,JonbesheSabz,obfs4
+154.35.22.12,1984,JonbesheSabz,obfs4
+154.35.22.12,1894,JonbesheSabz,obfs4
+154.35.22.12,4148,JonbesheSabz,obfs4
+154.35.22.12,4304,JonbesheSabz,obfs4
+154.35.22.12,13023,JonbesheSabz,obfs4
+154.35.22.12,26715,JonbesheSabz,obfs4
+154.35.22.12,26919,JonbesheSabz,obfs4
+154.35.22.12,34939,JonbesheSabz,obfs4
+154.35.22.12,36882,JonbesheSabz,obfs4
+154.35.22.12,40033,JonbesheSabz,obfs4
+154.35.22.12,44899,JonbesheSabz,obfs4
+154.35.22.12,22,JonbesheSabz,ssh
+154.35.22.13,80,Azadi,obfs4
+154.35.22.13,443,Azadi,obfs4
+154.35.22.13,1984,Azadi,obfs4
+154.35.22.13,4319,Azadi,obfs4
+154.35.22.13,6041,Azadi,obfs4
+154.35.22.13,16815,Azadi,obfs4
+154.35.22.13,17878,Azadi,obfs4
+154.35.22.13,29243,Azadi,obfs4
+154.35.22.13,30956,Azadi,obfs4
+154.35.22.13,39004,Azadi,obfs4
+154.35.22.13,50681,Azadi,obfs4
+154.35.22.13,59765,Azadi,obfs4
+154.35.22.13,62623,Azadi,obfs4
+154.35.22.13,22,Azadi,ssh
+192.95.36.142,443,Lisbeth,obfs4
+192.95.36.142,2200,Lisbeth,ssh
+85.17.30.79,22,NX01,ssh
+85.17.30.79,443,NX01,obfs4
+192.30.252.130,443,github.com,https
+192.30.252.130,444,github.com,port-444
+192.30.253.112,443,github.com,https
+192.30.253.112,444,github.com,port-444
```

Comment:
Yep, I see a reason to include `dport=444`. I just wanted to ensure that **dead** `192.30.252.130:443` is intentional too.

`github.com` does not resolve to `192.30.252.130` for me from three distinct vantage points in US, UK and RU.

`192.30.252.130` has PTR record pointing back to `github.com`, so it may be some old IP address for this name. The latest source saying that github.com points to 192.30.252.130 is dated January 2016. So it's unclear, is usage of stale IP address intentional or not.